integration/solana/simple.spec.ts

```diff
@@ -70,17 +70,17 @@ describe('Deploy solang contract and test', function () {
         res = await contract.functions.op_i256(0, 1000, 4100, { simulate: true });
         expect(Number(res.result)).toBe(5100);
         res = await contract.functions.op_i256(1, 1000, 4100, { simulate: true });
-        expect(Number(res.result)).toBe(-3100);
+        expect(res.result).toStrictEqual(BigInt(-3100));
         res = await contract.functions.op_i256(2, 1000, 4100, { simulate: true });
         expect(Number(res.result)).toBe(4100000);
         res = await contract.functions.op_i256(3, 1000, 10, { simulate: true });
         expect(Number(res.result)).toBe(100);
         res = await contract.functions.op_i256(4, 1000, 99, { simulate: true });
         expect(Number(res.result)).toBe(10);
         res = await contract.functions.op_i256(6, - 10000000000000, 8, { simulate: true });
-        expect(Number(res.result)).toBe(-2560000000000000);
+        expect(Number(res.result)).toBe(69551044457136128);
         res = await contract.functions.op_i256(7, - 10000000000000, 8, { simulate: true });
-        expect(Number(res.result)).toBe(-39062500000);
+        expect(Number(res.result)).toBe(1061264716448);
```

Comment:
@seanyoung These two results of shift were wrong. When I print the result in Solidity, I also get the new numbers. Have you checked if these numbers were indeed the correct output? If not, I believe the implementation of shift for `int256` is incorrect.

Reply:
The answers were correct:

`-10000000000000 << 8 = -2560000000000000`
`-10000000000000 >> 8 = -39062500000`

Somehow your change broke this.

Reply:
I'll investigate this first.

Reply:
Sorry, I was running the integration tests with an old version of `@solana/solidity`.

Reply:
Fixed